com.unity.render-pipelines.high-definition/Documentation~/snippets/shader-graph-blocks/iridescence-thickness.md

```diff
@@ -1,6 +1,6 @@
 <tr>
 <td>**Iridescence Thickness**</td>
-<td>The thickness of the iridescence. This changes the gradient of color that the iridescence effect produces.</td>
+<td>The thickness of the iridescence. This changes the gradient of color that the iridescence effect produces. Unit is micrometer multiply by 3. A value of 1 will be remap to 3 micrometer or 3000 nanometer.</td>
```

Comment:
Unit is micrometer multipl**ied** by 3. A value of 1 **is remapped** to 3 micrometer**s** or 3000 nanometer**s**